docs/getting-started.md

```diff
@@ -192,7 +188,10 @@ To do this we will simply call `Kill` on the task after waiting a couple of seco
 	}
 
 	status := <-exitStatusC
-	fmt.Printf("redis-server exited with status: %d\n", status)
+	if status.Err != nil {
```

Comment:
BTW, this is a suggestion to see what you all think.  I think its a common problem when chans return an error and the value, its easy to skip the error check